CinnVIIStarkMenu@NikoKrause/po/CinnVIIStarkMenu.pot

```diff
@@ -545,12 +545,12 @@ msgstr ""
 
 #. 3.2->settings-schema.json->search-filesystem->tooltip
 #. 2.6->settings-schema.json->search-filesystem->tooltip
-msgid "Allows path entry in the menu search box."
+msgid "Allows path entry into the menu search box."
 msgstr ""
 
 #. 3.2->settings-schema.json->show-shutdown-menu->description
 #. 2.6->settings-schema.json->show-shutdown-menu->description
-msgid "Show quit buttons"
+msgid "Show quit buttons on the sidebar"
```

Comment:
Ah yes. Since 3.0 or 3.2? it's possible to create Tabs in the settings.
As it is right now, it's all in one place. It's sometimes difficult to decide, where to push, which settings. I hope it's better in the new version I did, not pushed yet.